README.md

```diff
@@ -166,6 +166,27 @@ Note that the following will fail:
 Use the IsNil checker below for this kind of nil check.
 
 
+### ErrorAs
+
+ErrorAs is a Checker checking that the error is or wraps the provided error and
+if so, assigns it to the pointer. This is analogous to calling errors.As.
+
+For instance:
+
+    var pathError *os.PathError
+    c.Assert(err, qt.ErrorAs, &pathError)
```

Comment:
I think it would be worth adding here, in addition to this example, the other you provided in the pr description:
```
var customError CustomError
if c.Check(err, qt.ErrorAs, &customError) {
    c.Assert(customError.Something, …)
}
```
What do you think?

Reply:
Yeah, I only realized this as I was posting the PR. I should have gone back and added it to the docs.

I've added an example that conveys the point even better than this example. Let me know what you think!